.gitattributes

```diff
@@ -3,6 +3,7 @@
 /.github/ export-ignore
 /.gitignore export-ignore
 /.phive/ export-ignore
+/.php-cs-fixer.dist.php export-ignore
```

Comment:
Now would maybe be an appropriate time to group all the config files for checking tools into a common folder so that the root folder doesn’t become a mess.

Reply:
Done: #288

I'll update this PR once the other PR is merged.

Should we move `phpunit.xml` as well?

Reply:
And should we also move the `Doxyfile`?

Reply:
Yes, Doxyfile would be great. I’m on the fence about phpunit.xml. I kinda like the fact that you can just run phpunit without args in the root dir. But I think I like clean dirs even more… WDYT?

Reply:
Okay, then I'll move `Doxyfile` as well. Concerning `phpunit.xml`: I'd also prefer to keep it in the project root as  this makes tool integration a lot easier (e.g., being able to easily run the tests in PhpStorm).

Reply:
Concerning the config file move: Done, rebased and repushed.